app/build.gradle

```diff
@@ -92,7 +93,7 @@ dependencies {
     implementation 'com.google.firebase:firebase-messaging:20.2.4'
 
     // Coroutine play services
-    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-play-services:1.1.1'
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-play-services:1.3.7'
 
     // ThreeTenBP
     implementation 'com.jakewharton.threetenabp:threetenabp:1.2.4'
```

Comment:
cài này dùng lm gì thế ô ???

Reply:
Cái này khi dùng firebase thay vì dùng callback addOnSuccessListener{} hay addOnFailedListener{} thì chỉ cần .await() theo kiểu coroutines thôi

Reply:
thừa , t toàn dùng await có cần thêm thư viện threetenapbp kia đâu .

Reply:
:)) t tưởng ô hỏi cái kotlinx-coroutines-play-services
LocalDateTime requires API level 26 nên threetenapbp để suport API < 26

Còn cái kotlinx-coroutines-play-services không thừa đâu. Nó là Task<T>.await()

Reply:
ok